design/user_types.go

```diff
@@ -143,3 +208,10 @@ var identityData = a.Type("IdentityData", func() {
 	a.Attribute("attributes", identityDataAttributes, "Attributes of the user identity")
 	a.Required("type", "attributes")
 })
+
+// WorkItemLinks has `self` as of now according to http://jsonapi.org/format/#fetching-resources
+var WorkItemLinks = a.Type("WorkItemLinks", func() {
+	a.Attribute("self", d.String, func() {
+		a.Example("http://localhost:8080/api/workitems.2/1")
```

Comment:
Maybe it is better for documentation if the example URL contains `api.almighty.io` rather than `localhost:8080`, don't you think?